### DISCUSSION TITLE
Light dimmer with zero-cross

### DISCUSSION THREAD

**Discussion by lukaszwitkowski** [RavenSystem/esp-homekit-devices#1690] [MEPLHAA Scripts]
Do you have code to use dimmer module with zero-crossing detection?
![image](https://user-images.githubusercontent.com/36861888/149015483-226419a9-9635-41c5-8dd9-8e6caa9ba19e.jpeg)

**Comment by jmgutierrezc1** [comment]
I am also interested in this....but have not being able to do it

**Comment by xrust83** [comment]
{"c":{"io":[[[3],1],[[2],7,0,0,2],[[0],6]],
"q":110,"zc":[3,2]},
"a":[{"t":30,"g":[2],
"b":[{"g":0}],"e":5,"d":0.2,"st":384}]}

**Comment by jmgutierrezc1** [comment]
Thank you very much, it works!, I will try to seek adjustment as bulb truns off at 43%....
